apps/src/music/blockly/blocks/simple2.js

```diff
@@ -239,3 +239,71 @@ export const playSoundsSequential = {
       ProgramSequencer.endSequential();
       `
 };
+
+export const repeatSimple2 = {
+  definition: {
+    type: BlockTypes.REPEAT_SIMPLE2,
+    message0: 'repeat %1 times',
+    args0: [
+      {
+        type: 'input_value',
+        name: 'times'
+      }
+    ],
+    message1: 'do %1',
+    args1: [
+      {
+        type: 'input_statement',
+        name: 'code'
+      }
+    ],
+    inputsInline: true,
+    previousStatement: null,
+    nextStatement: null,
+    style: 'loop_blocks',
+    tooltip: 'repeat',
+    helpUrl: ''
+  },
+  generator: block => {
+    const repeats =
+      Blockly.JavaScript.valueToCode(
+        block,
+        'times',
+        Blockly.JavaScript.ORDER_ASSIGNMENT
+      ) || 0;
+
+    let branch = Blockly.JavaScript.statementToCode(block, 'code');
+    branch = Blockly.JavaScript.addLoopTrap(branch, block);
```

Comment:
It's possible we should be adding this to more of our generated code.

This showed up when adapting the generator from Blockly's [implementation](https://github.com/google/blockly/blob/29acad04d3123a6d4df176bf03f7fe5e0b2c0ac7/generators/javascript/loops.js#L19-L45).